docs/source/notebooks/Subset_climate_data_over_watershed.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
Second cell fails for me with:
HTTPConnectionPool(host='localhost', port=8093): Max retries exceeded with url: /?service=WPS&request=GetCapabilities&version=1.0.0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb055842668>: Failed to establish a new connection: [Errno 111] Connection refused',))

Hopefully just an artefact of my setup and not something more sinister.

Reply:
You probably did not launch Flyingpigeon.